builtin/providers/azurerm/resource_arm_template_deployment.go

```diff
@@ -155,20 +155,44 @@ func resourceArmTemplateDeploymentRead(d *schema.ResourceData, meta interface{})
 	if resp.Properties.Outputs != nil && len(*resp.Properties.Outputs) > 0 {
 		outputs = make(map[string]string)
 		for key, output := range *resp.Properties.Outputs {
+			log.Printf("[DEBUG] Processing deployment output %s", key)
 			outputMap := output.(map[string]interface{})
 			outputValue, ok := outputMap["value"]
 			if !ok {
 				// No value
 				continue
 			}
+			outputType := outputMap["type"]
```

Comment:
Since these come straight from Azure, and it does its own type checks, I've removed those checks in my code.